### DISCUSSION TITLE
停贷的家人，我们江西目前各大银行，出台了各种宽限政策给到业主，签完协议，可以免除现在的征信罚息等

### DISCUSSION THREAD

**Discussion by honghaicheng** [WeNeedHome/SummaryOfLoanSuspension#1194] [Ideas]
想问一下，各位其他地方，其他楼盘的烂兄烂弟们，你们银行有出面跟你们沟通怎么解决这个事情吗？
然后宽限协议能跟银行签吗？

**Comment by honghaicheng** [comment]
help

**Comment by Cegazhos** [comment]
问问律师吧。
私以为不该签。本来集体停贷就是参考那些预售房烂尾了交易取消的方向走的，那你签了这个协议岂不是一种变相地承认集体停贷是违法违规的么?
感觉还是太软弱了 挥一挥棒子就服软了(

**Comment by myserillibel** [comment]
这本身就是完全合理合法的

**Comment by ytx0574** [comment]
我的不是， 银行打电话说现在还了之前停贷的， 可以享受半年后再还。  利息一分不少。
